### PR TITLE
[improvement](detailMessage) add AvailCapacity prompt in detailMessage

### LINKED ISSUE #26253: [Bug] failed to  truncate table
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/doris/issues?q=is%3Aissue) and found no similar issues.

### Version

2.0.2

### What's Wrong?

1. fe 执行truncate 失败了
MySQL [test_db]> truncate table create_table_use_created_policy;
ERROR 1105 (HY000): errCode = 2, detailMessage = Failed to find enough backend, please check the replication num,replication tag and storage medium.
Create failed replications:
replication tag: {"location" : "default"}, replication num: 3, storage medium: SSD

2.
create_table_use_created_policy 是一个冷热存储分离的表

MySQL [test_db]> show create table create_table_use_created_policy\G;
*************************** 1. row ***************************
       Table: create_table_use_created_policy
Create Table: CREATE TABLE `create_table_use_created_policy` (
  `k1` bigint(20) NULL,
  `k2` largeint(40) NULL,
  `v1` varchar(2048) NULL
) ENGINE=OLAP
UNIQUE KEY(`k1`)
COMMENT 'OLAP'
DISTRIBUTED BY HASH(`k1`) BUCKETS 3
PROPERTIES (
"replication_allocation" = "tag.location.default: 3",
"is_being_synced" = "false",
"storage_format" = "V2",
"light_schema_change" = "true",
"storage_policy" = "test_policy",
"disable_auto_compaction" = "false",
"enable_single_replica_compaction" = "false"
);

MySQL [test_db]> show STORAGE POLICY\G;
*************************** 1. row ***************************
      PolicyName: test_policy
              Id: 12025
         Version: 0
            Type: STORAGE
 StorageResource: remote_s3
CooldownDatetime: -1
     CooldownTtl: 3600
1 row in set (0.00 sec)

### What You Expected?

像是storage policy没了

### How to Reproduce?

https://doris.apache.org/zh-CN/docs/advanced/cold_hot_separation
根据官网建表，然后数据降冷后执行查询，最后执行truncate

### Anything Else?

无

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by xingyingone**
## Proposed changes

Issue Number: close #26253 

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by xingyingone** [comment]
run buildall

**Review by alexxing662 — APPROVED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/system/SystemInfoService.java [hunk 1]
```diff
@@ -593,7 +593,7 @@ public Map<Tag, List<Long>> selectBackendIdsForReplicaCreation(
             if (!failedEntries.isEmpty()) {
                 String failedMsg = Joiner.on("\n").join(failedEntries);
                 throw new DdlException("Failed to find enough backend, please check the replication num,"
-                        + "replication tag and storage medium.\n" + "Create failed replications:\n" + failedMsg);
+                        + "replication tag and storage medium and AvailCapacity.\n" + "Create failed replications:\n" + failedMsg);
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/system/SystemInfoService.java:596]:
```suggestion
                        + "replication tag and storage medium and avail capacity of backends.\n" + "Create failed replications:\n" + failedMsg);
```

**Comment by xingyingone** [comment]
run buildall

**Review by morningman — DISMISSED** [review]
LGTM

**Comment by xingyingone** [comment]
run buildall

**Comment by xingyingone** [comment]
run buildall

**Comment by xingyingone** [comment]
run buildall

**Comment by xingyingone** [comment]
run buildall

**Comment by xingyingone** [comment]
run buildall

**Comment by xingyingone** [comment]
run buildall

**Comment by xingyingone** [comment]
run buildall

**Comment by xingyingone** [comment]
run buildall  tpch-sf100 (tpch)

**Comment by xingyingone** [comment]
run tpch-sf10

**Comment by xingyingone** [comment]
run buildall

**Comment by xingyingone** [comment]
run buildall

**Comment by xingyingone** [comment]
run buildall

**Comment by xingyingone** [comment]
run buildall

**Comment by xingyingone** [comment]
run buildall

**Comment by xingyingone** [comment]
run buildall

**Comment by xingyingone** [comment]
> Could you put an example of output?

before modification:
MySQL [test_db]> truncate table create_table_use_created_policy;
ERROR 1105 (HY000): errCode = 2, detailMessage = Failed to find enough backend, please check the replication num,replication tag and storage medium.
Create failed replications:
replication tag: {"location" : "default"}, replication num: 3, storage medium: SSD


after modification:
MySQL [test_db]> truncate table create_table_use_created_policy;
ERROR 1105 (HY000): errCode = 2, detailMessage = Failed to find enough backend, please check the replication num,replication tag and storage medium and avail capacity of backends.
Create failed replications:
replication tag: {"location" : "default"}, replication num: 3, storage medium: SSD

**Comment by xingyingone** [comment]
@dataroaring

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
LGTM

**Comment by yiguolei** [comment]
run buildall

**Comment by xingyingone** [comment]
run feut

**Pull request merged**